### PR TITLE
Docs: Mobile Toast docs

### PR DESCRIPTION
**Pull request by ponori**
Adding Android/iOS Toast docs

### CODE REVIEW

File: docs/markdown/ios/toast.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    Leave toasts on screen for a long time without a way to dismiss. Exceptions are blocking error toasts where a user can’t take any action until the error is resolved.
+  </Group>
+</TwoCol>
+
+## Accessibility
+
+People use Apple’s accessibility features, such as reduced transparency, VoiceOver, and increased text size to personalize how they interact with their device. Supporting these personalizations ensures that everyone has a great user experience. See Apple’s Human Interface Guidelines and documentation about accessibility for iOS:
+
+[Accessbile design on Android](https://material.io/design/usability/accessibility.html#understanding-accessibility)
+[Accessible development on Android](https://developer.android.com/guide/topics/ui/accessibility)
```
Comment (ayeshakmaz) [inline docs/markdown/ios/toast.md:68]: these should say iOS

File: docs/markdown/ios/toast.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+### Duration
+Some people may take longer to read toasts than others due to aging or other cognitive impairments Use the guide below to set duration for Toasts:
```
Comment (ayeshakmaz) [inline docs/markdown/ios/toast.md:71]: Missing period before "Use the guide" . Also maybe:

Some people may take longer to read toasts than others due to aging, low vision, or cognitive impairments.

File: docs/markdown/ios/toast.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+- Brief text of approximately 10–15 words (including button text): 5s
+- Longer than 15 words: Slow readers can read about 125–200 words per minute. Base your duration on the slowest number. For example, a toast with 20 words should be set to 10s. [Source](https://capitalizemytitle.com/reading-time/3000-words/)
```
Comment (ayeshakmaz) [inline docs/markdown/ios/toast.md:74]: I would spell out seconds instead of just the s

File: docs/markdown/ios/toast.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+<TwoCol>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/02/40/85/024085af164541554f69c18368d7078e.jpg" noPadding alt="text only toast"/>
+    
+    **Text only**
+    A simple, generic acknowledgment after an action is taken These should not be actionable.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/02/40/85/024085af164541554f69c18368d7078e.jpg" noPadding alt="toast with image"/>
```
Comment (ayeshakmaz) [inline docs/markdown/ios/toast.md:106]: I think this is the wrong image, it has an image :P

File: docs/markdown/ios/toast.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+    **Image**
+    With an image for Pin or Board actions.
+  </Group>
+</TwoCol>
+<TwoCol>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/fa/1d/e5/fa1de5fbe15b0a3ee774f40c0a46b66c.jpg" noPadding alt="toast with avatar"/>
+    
+    **Avatar**
+    With an Avatar for Profile or Pinner-related messaging.  An optional link be included. When there’s a link on mWeb, the entire toast is tapable, using tapArea.
```
Comment (ayeshakmaz) [inline docs/markdown/ios/toast.md:117]: " With an Avatar for Profile or Pinner-related messaging.  An optional link **can** be included. When there’s a link on mWeb, the entire toast is **tappable**, using **TapArea**."

File: docs/markdown/ios/toast.md [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+    <ImgContainer src="https://i.pinimg.com/originals/02/40/85/024085af164541554f69c18368d7078e.jpg" noPadding alt="toast with link"/>
+    
+    **Link**
+    As a secondary element, to drive users to another surface.
+  </Group>
+</TwoCol>
+
+### Placement
+
+Placement is always centered at the bottom of the screen and not blocking any navigation or important buttons.
```
Comment (ayeshakmaz) [inline docs/markdown/ios/toast.md:143]: All the examples show it centered at the top of the screen?

File: docs/markdown/ios/toast.md [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+<TwoCol>
+  <Group>
+    <Do title="When to use" />      
+  - Consider internationalization and how other languages may be take up more space
+  - Be brief and concise
+  - Use conversational language
+  </Group>
+  <Group>
+  <Dont title="When not to use" />
+  - Use lengthy, technical jargon or local idioms that will be hard to translate to other languages
```
Comment (ayeshakmaz) [inline docs/markdown/ios/toast.md:156]: Titles here should be "Do" and "Don't"

File: docs/markdown/ios/toast.md [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+fullwidth: true
+---
+
+<ImgContainer src="https://i.pinimg.com/originals/27/32/0d/27320d8b7731c7cea23fbf917e7d2160.jpg" noPadding alt="an example of toast"/>
+
+## Usage guidelines
+
+<TwoCol>
+  <Group>
+    <Do title="When to use" />
```
Comment (ayeshakmaz) [inline docs/markdown/ios/toast.md:13]: Double check that these all have periods at the end

File: docs/markdown/ios/toast.md [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+
+<ImgContainer src="https://i.pinimg.com/originals/27/32/0d/27320d8b7731c7cea23fbf917e7d2160.jpg" noPadding alt="an example of toast"/>
+
+## Usage guidelines
+
+<TwoCol>
+  <Group>
+    <Do title="When to use" />    
+    - Briefly acknowledging a user action without interrupting their flow
+    - When acknowledging an action that relates to another surface, providing a link that navigates the user to that surface
```
Comment (ayeshakmaz) [inline docs/markdown/ios/toast.md:15]: provide instead of providing

File: docs/markdown/ios/toast.md [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+    - To undo actions after acknowledgement, if there isn’t already a way to do so on the current surface
+    - For system processes like showing that a process is loading, or when there are internet connectivity issues.
+  </Group>
+  <Group>
+  <Dont title="When not to use" />
+  - When, due to an error, a user can’t even continue performing basic tasks like browsing already loaded Pins.
+  - When asking a user to confirm that they want to perform an action. Use [Sheet](/ios/sheet) instead.
+  - When you want to suggest a user spend more money or try new features; use [Upsell](/upsell) instead.
+  - For errors that relate to a specific section or page. Use [Callout](/web/callout) or [SlimBanner](/slimbanner) instead.
+  - To guide or educate the user. Use [Popover](/popover) or [Tooltip](/tooltip) instead.
```
Comment (ayeshakmaz) [inline docs/markdown/ios/toast.md:25]: make sure these all have /web in front of them

File: docs/markdown/android/toast.md [hunk 11]
```diff
@@ -0,0 +1,158 @@
...
+    <ImgContainer src="https://i.pinimg.com/originals/32/66/29/326629104e4376649ed9c910516423c2.jpg" alt="example of leaving a toast on screen"/>
+    <Dont title="Don't" />
+    Leave toasts on screen for a long time without a way to dismiss. Exceptions are blocking error toasts where a user can’t take any action until the error is resolved.
+  </Group>
+</TwoCol>
+
+## Accessibility
+
+People use Apple’s accessibility features, such as reduced transparency, VoiceOver, and increased text size to personalize how they interact with their device. Supporting these personalizations ensures that everyone has a great user experience. See Apple’s Human Interface Guidelines and documentation about accessibility for iOS:
+
```
Comment (ayeshakmaz) [inline docs/markdown/android/toast.md:66]: this all needs to be updated for Android

File: docs/markdown/ios/toast.md (hunk 1 above)
Comment (ponori) [reply docs/markdown/ios/toast.md:68]: Done

File: docs/markdown/ios/toast.md (hunk 2 above)
Comment (ponori) [reply docs/markdown/ios/toast.md:71]: Nice! Done!

File: docs/markdown/ios/toast.md (hunk 3 above)
Comment (ponori) [reply docs/markdown/ios/toast.md:74]: Done!

File: docs/markdown/ios/toast.md (hunk 4 above)
Comment (ponori) [reply docs/markdown/ios/toast.md:106]: Fixed, thanks!

File: docs/markdown/ios/toast.md (hunk 5 above)
Comment (ponori) [reply docs/markdown/ios/toast.md:117]: Done

File: docs/markdown/ios/toast.md (hunk 8 above)
Comment (ponori) [reply docs/markdown/ios/toast.md:13]: Done

File: docs/markdown/ios/toast.md (hunk 9 above)
Comment (ponori) [reply docs/markdown/ios/toast.md:15]: Done

File: docs/markdown/ios/toast.md (hunk 10 above)
Comment (ponori) [reply docs/markdown/ios/toast.md:25]: Done

File: docs/markdown/android/toast.md (hunk 11 above)
Comment (ponori) [reply docs/markdown/android/toast.md]: Done

**Review by ayeshakmaz — APPROVED** [review]

**Pull request merged**